content/pricing/billing.textile

```diff
@@ -0,0 +1,35 @@
+---
+title: Billing
+meta_description: "Understand how invoicing and billing works for Ably packages."
+---
+
+Your monthly bills are calculated based on the base package fee, plus the number of messages, and channel and connection minutes that you used for that month.
+
+Consumption is calculated on the last day of the month and invoices are issued in arrears on the 1st of the following month. Note that for consumption, minutes are the smallest unit that we bill against. So if a connection was open for 10.01 minutes then this will be billed as 11 minutes of usage.
+
+h2(#package-changes). Upgrades and downgrades
+
+For package upgrades, the base package price will be charged pro-rata from the point in the month that you upgraded. This is true for both base package prices if you are upgrading from a paid package.
+
+Package downgrades take effect on the 1st of the following month.
+
+<aside data-type='note'>
+<p>If you have an "Enterprise package":/pricing/enterprise with Ably, then contact your Customer Success Manager to discuss any details or queries you may have regarding your package.</p>
+</aside>
+
+h2(#billing). Billing details and invoices
```

Comment:
Probably not in this but in a follow up PR we should add note re organizations (that billing is configured for each account separately)